roles/openshift_service_catalog/templates/controller_manager.j2

```diff
@@ -46,6 +46,14 @@ spec:
 {% if openshift_service_catalog_async_bindings_enabled | bool %}
         - --feature-gates
         - AsyncBindingOperations=true
+{% endif %}
+{% if openshift_service_catalog_namespaced_service_brokers_enabled | bool %}
+        - --feature-gates
+        - NamespacedServiceBroker=true
+{% endif %}
+{% if openshift_service_catalog_catalog_restrictions_enabled | bool %}
```

Comment:
I think it's fine to just have this on by default. I don't think it should be behind a feature gate.